### PR TITLE
Fix EZP-23707: Remove dependency on legacy kernel for system information page

### PR DESCRIPTION
**Pull request by lolautruche**
> https://jira.ez.no/browse/EZP-23707

### CODE REVIEW

**Comment by lolautruche** [comment]
Review ping @andrerom @bdunogier @dpobel

**Comment by andrerom** [comment]
missing some template changes or?

**Comment by lolautruche** [comment]
Updated thanks.

As for version to display, I created a follow-up issue: [EZP-23932](https://jira.ez.no/browse/EZP-23932).

**Comment by andrerom** [comment]
+1

File: Helper/SystemInfoHelper.php [hunk 1]
```diff
@@ -32,16 +27,15 @@ class SystemInfoHelper implements SystemInfoHelperInterface
...
-     * The database connection, only used to retrieve some information on the 
+     * The database connection, only used to retrieve some information on the
      * database itself.
      *
      * @var \Doctrine\DBAL\Connection
      */
     private $connection;
 
-    public function __construct( \Closure $legacyKernelClosure, Connection $db, array $bundles )
+    public function __construct(Connection $db, array $bundles )
```
Comment (pspanja) [inline Helper/SystemInfoHelper.php]: CS: space after opening parenthesis.

**Comment by pspanja** [comment]
+1

File: Helper/SystemInfoHelper.php (hunk 1 above)
Comment (lolautruche) [inline Helper/SystemInfoHelper.php]: Thanks for that.

**Comment by lolautruche** [comment]
Still waiting for @dpobel :wink:

**Comment by dpobel** [comment]
+1 (but Still waiting for Travis :))

**Pull request merged**